### ISSUE TITLE
find a solution/replacement for MultiAgents

### ISSUE THREAD

**Issue by mhinsch** [MRC-CSO-SPHSU/LoneParentsModel.jl#22]
Currently some general functionality is still imported from MultiAgents. If LoneParentsModel.jl is supposed to be completely independent of MultiAgents.jl these imports should be removed. It would be nice to find a solution that doesn't duplicate effort, though.

**Comment by AtiyahElsheikh** [comment]
Indeed, LPM.jl is currently making use of  MultiAgents.jl. Or may be I am getting you wrong?
